src/components/composer/top_bar_composer/top_bar_composer.xml

```diff
@@ -5,6 +5,7 @@
         focus="props.focus"
         inputStyle="composerStyle"
         onComposerContentFocused="props.onComposerContentFocused"
+        placeholder="placeholderText"
```

Comment:
you can just hardcode the string here
```suggestion
        placeholder="'fx'"
```

Reply:
Thank you for your input. I have made the changes.